docker-compose.yml

```diff
@@ -24,3 +30,31 @@ services:
         dockerfile: Dockerfile.worker
     ports:
       - "8181:8181"
+
+  timescaledb:
+    container_name: timescaledb
+    image: timescale/timescaledb:latest-pg10
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_PASSWORD=password
+    volumes:
+      - timescaledb:/var/lib/postgresql
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  jaeger:
+    container_name: jaeger
+    image: jaegertracing/all-in-one:1.19
+    ports:
+      - "6831:6831/udp"
+      - "5778:5778"
+      - "16686:16686"
+
+volumes:
+  timescaledb:
+  grafana-data:
```

Comment:
Did you intend to include the grafana volume here?

Reply:
Nope. Fixed